### ISSUE TITLE
Precompilation error under julia v1.9.0-rc2

### ISSUE THREAD

**Issue by JoshuaLampert** [ranocha/SummationByPartsOperators.jl#175]
With julia v1.9.0-rc2 I get the following error during precompilation:

```julia
ERROR: LoadError: MethodError: no method matching convolve_interior_coefficients!(::Vector{Float64}, ::StaticArraysCore.SVector{1, Float64}, ::Float64, ::StaticArraysCore.SVector{1, Float64}, ::Vector{Float64}, ::Float64, ::Int64, ::Int64, ::SummationByPartsOperators.FastMode)

Closest candidates are:
  convolve_interior_coefficients!(::AbstractVector, ::StaticArraysCore.SVector{LowerOffset}, ::Any, ::StaticArraysCore.SVector{UpperOffset}, ::AbstractVector, ::Any, ::Any, ::Static.StaticInt{left_boundary_width}, ::Static.StaticInt{right_boundary_width}, ::Any) where {LowerOffset, UpperOffset, left_boundary_width, right_boundary_width}
[...]
```

**Comment by ranocha** [comment]
I filed an issue at the main Julia repo: https://github.com/JuliaLang/julia/issues/49250

**Comment by JoshuaLampert** [comment]
I can confirm that this is fixed under julia v1.9.0-rc3.
